src/Flicking.ts

```diff
@@ -540,6 +541,76 @@ class Flicking extends Component {
     return this.viewport.remove(index, deleteCount);
   }
 
+  /**
+   * Synchronize info of panels instance with info given by external rendering.
+   * @ko 외부 렌더링 방식에 의해 입력받은 패널의 정보와 현재 플리킹이 갖는 패널 정보를 동기화한다.
+   * @param diffInfo - Info object of how panel elements are changed.<ko>패널의 DOM 요소들의 변경 정보를 담는 오브젝트.</ko>
+   * @param {HTMLElement[]} [diffInfo.list] - DOM elements list after update.<ko>업데이트 이후 DOM 요소들의 리스트</ko>
+   * @param {number[][]} [diffInfo.maintained] - Index tuple array of DOM elements maintained. Formatted with `[before, after]`.<ko>변경 전후에 유지된 DOM 요소들의 인덱스 튜플 배열. `[이전, 이후]`의 형식을 갖고 있어야 한다.</ko>
+   * @param {number[]} [diffInfo.added] - Index array of DOM elements added to `list`.<ko>`list`에서 추가된 DOM 요소들의 인덱스 배열.</ko>
+   */
+  public sync(diffInfo: {
+    list: HTMLElement[],
+    maintained: number[][],
+    added: number[],
+  }) {
+    const { list, added, maintained } = diffInfo;
+
+    const viewport = this.viewport;
+    const panelManager = viewport.panelManager;
+    const indexRange = panelManager.getRange();
+    const isCircular = this.options.circular;
+
+    // Make sure that new "list" should include cloned elements
+    const originalPanelCount = (list.length / (panelManager.getCloneCount() + 1)) >> 0; // Make sure it's integer. Same with Math.floor, but faster
+    const shouldCloneCount = ((list.length / originalPanelCount) >> 0) - 1;
```

Comment:
@WoodNeck What's the meaning of `should` in `shouldCloneCount`?

Reply:
It can interpreted to `given`.
As `list` is given, we should set cloneCount to that value.

Reply:
How about `newCloneCount`?

Reply:
That can be more nice, I'll fix it in next commit.